includes/Media/Video/Poster.php

```diff
@@ -235,6 +235,11 @@ public function get_thumbnail_data( int $thumbnail_id ): array {
 	 * @return void
 	 */
 	public function delete_video_poster( int $attachment_id ) {
+		/**
+		 * Post ID.
+		 *
+		 * @var int $post_id
```

Comment:
Should this also be false?

Reply:
Just updated the PHPDoc because the result of `get_post_meta` is either `int 0` or a non-empty string.